extensions/amp-a4a/amp-a4a-format.md

```diff
@@ -61,6 +61,19 @@ rel="canonical">` tag.
    and won't be independently search-indexed, so self-referencing
    would be useless.
 
+1. The creative can include optional meta tags in HTML head as identifiers,
+   in the format of `<meta name="amp4ads-id" content="${namespace}:${id}">`.
```

Comment:
Suggestion: Maybe provide a regex or other constraint on the forms of `${namespace}` and `${id}`, just to prevent people from doing stuff like `namespace = '"::"'` or `id = '"><script>doSomethingEvil()</script>'`.  Some of those are blocked by the validator, of course, but some would just make everybody's life painful to no good reason.

The RE would have to be flexible enough to allow ad networks to have their own ID space.  Something like `[a-zA-Z0-9_-/.?%]` maybe?  Or just something that excludes quotes and `:` maybe.

Come to think of it.  If `meta` tags allow arbitrary attributes, maybe it would be better to split into two fields so that we don't have to worry about `:` at all:

```html
<meta name="amp4ads-id" namespace="${namespace}" content="${id}">
```